src/main/java/com/easypost/service/BatchService.java

```diff
@@ -198,13 +199,13 @@ public Batch createScanForm(final String id) throws EasyPostException {
     /**
      * Create a scan form for this batch.
      *
-     * @param id The ID of batch.
+     * @param id     The ID of batch.
      * @param params Map of parameters.
```

Comment:
Some docstrings got unnecessarily formatted

Reply:
This is correct format to make params align each other

Reply:
VS code auto format/correct it